src/graph.rs

```diff
@@ -936,14 +933,15 @@ fn branch_color<T: Clone>(
     unknown: &[T],
     counter: usize,
 ) -> T {
-    let color = order
-        .iter()
-        .find_position(|(b, _)| {
-            (name.starts_with(ORIGIN) && b.is_match(&name[7..])) || b.is_match(name)
-        })
-        .map(|(_pos, col)| &col.1[counter % col.1.len()])
-        .unwrap_or_else(|| &unknown[counter % unknown.len()]);
-    color.clone()
+    let stripped_name = name.strip_prefix("origin/").unwrap_or(name);
```

Comment:
Maybe stay with using the string constant `ORIGIN` here?

Reply:
Good point. I will update the code.